F1WM/Controllers/NewsController.cs

```diff
@@ -49,7 +49,100 @@ public async Task<IActionResult> GetSingle(int id)
 			}
 		}
 
-		[HttpGet("important")]
+        [HttpGet("tag/{id}")]
+        [Produces("application/json", Type = typeof(NewsSummary))]
+        public async Task<IActionResult> GetByTag(int id)
+        {
+            try
+            {
+                var news = await service.GetNewsByTag(id);
+                return (news.Any() ? (IActionResult)Ok(news) : (IActionResult)NotFound());
```

Comment:
Just to be consistent with other endpoints, please change the logic to return an array of items, no matter if there are any or not. See other news endpoints for examples.